src/Core/CHANGELOG.md

```diff
@@ -2,13 +2,20 @@
 
 ## NOT RELEASED
 
+## 1.9.0
+
 ### Added
 
 - Changed case of object's properties to camelCase.
 - Added documentation in class's headers.
 - Removed `final` from `ClientException` and `ServerException`.
 - Make Responses thrown Business Exception when AwsErrorCode <-> Exception class mapping provided through RequestContext.
 - Added Business Exceptions.
```

Comment:
Do you mean "domain exceptions"?

Reply:
yeah. I was thinking about business logic...